ipaclient/install/client.py

```diff
@@ -2345,6 +2345,75 @@ def update_ipa_nssdb():
                                    (nickname, sys_db.secdir, e))
 
 
+def sync_time(options, fstore, statestore, force):
+    """
+    Will disable any other time synchronization service if there is
+    --force-chrony option set, and configure chrony with given ntp(chrony)
+    server and/or pool using Augeas in configure_chrony method.
```

Comment:
Will disable any other time synchronization service if the --force-chrony option is set

Reply:
Oki